### DISCUSSION TITLE
ERROR 205: invalid keyword with *.inp file

### DISCUSSION THREAD

**Discussion by JuanGuerrero09** [pyswmm/pyswmm#519] [Q&A]
Hello, I was trying to use PYSWWM in conjunction with WATERGEMS, since it's possible to export the *.inp file.

I started exporting it to EPANET, and gave a .inp file really similar to the one in the examples, however, the main difference is that instead of having [CONDUITS] I have [PIPES], that also have the section information, instead of having it in a [XSECTIONS] part, and so on.

Is the .inp file for PYSWMM different from the one that [EPANET receives](http://wateranalytics.org/EPANET/_inp_file.html)?

**Comment by bemcdonnell** [comment]
@JuanGuerrero09 

https://www.epa.gov/system/files/documents/2022-04/swmm-users-manual-version-5.2.pdf

page 324 and onward has the SWMM file formats / section information
